activiti-cloud-modeling-service/activiti-cloud-acceptance-tests-modeling/src/main/java/org/activiti/cloud/acc/modeling/modeling/ModelingNamingIdentifier.java

```diff
@@ -50,7 +50,7 @@ public static ModelingNamingIdentifier<Project> projectNamed(String name) {
         return new ModelingNamingIdentifier<Project>(name) {
             @Override
             protected String getName(Project project) {
-                return project.getName();
+                return project.getTechnicalName();
```

Comment:
The step to renaming the getTechnicalName that is not the final name yet should be done in the end Why it's already here?